resources/views/admin/users/show.blade.php

```diff
@@ -9,7 +9,103 @@
+              <div class="font-weight-bold mb-0">{{$user->username}}</div>
+              <div >{{$user->email}}</div>
+              <br>
+              <div>{{$user->address}}</div>
+              <div>{{$user->city}}, {{$user->state}} {{$user->postal}}</div>
+              <div>{{$user->country}}</div>
+              <div>{{$user->phone}}</div>
+              <br>
+              <div>Default Time Zone: <span class="font-weight-bold"> {{$user->timezone}} </span></div>
+              @if ($user->language == "us_en")
+              <div>Language: <span class="font-weight-bold"> English </span></div>
+              @else
+              <div>Language: <span class="font-weight-bold"> {{$user->language}} </span></div>
+              @endif
+              <br>
+              <div>Created: <span class="font-weight-bold"> {{$user->created_at->format( 'm/j/Y g:i A')}} </span></div>
+              <div>Updated: <span class="font-weight-bold"> {{$user->updated_at->format( 'm/j/Y g:i A' )}} </span></div>
+              @if ($user->loggedin_at != null)
+              <div>Last Login: <span class="font-weight-bold"> {{$user->loggedin_at->format('m/j/Y g:i A')}} </span></div>
+              @else
+              
+              @endif
+              <br>
+            </div>
+            <div class="col" align="right">
+              <button class="btn btn-outline-secondary"> <i class="fas fa-lock"></i> permissions</button>
+              <button class="btn btn-secondary"> <i class="fas fa-edit"></i> edit</button>
+              <button class="btn btn-secondary"> <i class="fas fa-trash-alt"></i> delete</button>
+            </div>
+          </div>
+          <br>
+          <br>
+          <div class="row">
+            <div class="col align-self-center">
+              <h3 class="m-0">Groups</h3>
+            </div>
+            <div class="col form-inline input-group justify-content-end">
+              <div class="input-group-prepend">
+                <span class="input-group-text" id="prepend-search"><i class="fa fa-search" aria-hidden="true"></i></span>
+              </div>
+              <input type="text" class="form-control w-50 border-left-0" placeholder="Search">
+              <button type="submit" class="btn btn-secondary ml-2"> <i class="fas fa-plus"></i> Group</button>
+            </div>
+           </div>
+          <table class="table vuetable">
+          <thead>
+            <tr>
+              <th scope="col" class="text-uppercase">Full Name</th>
+              <th scope="col" class="text-uppercase">Description</th>
+              <th scope="col"></th>
+            </tr>
+          </thead>
+          <tbody>
+            @forelse ($user->members() as $group_member)
+            <tr>
+              <td>{{$group_member->group->name}}</td>
+              <td>{{$group_member->group->status}}</td>
+              <td class="text-right"><i class="fas fa-trash-alt text-secondary"></i></td>
+            </tr>
+            @empty
```

Comment:
Remove the groups table entirely if there are not groups found. Put it inside of a card with the text "User is not a member of any groups" centered with padding (See Zeplin).